libs/menu/src/DropdownMenu.tsx

```diff
@@ -133,7 +133,11 @@ function DropdownMenu({
   const responsiveIcon = (
     <div>
       <DropdownContext.Consumer>
-        {({ isExpanded }) => (isExpanded ? finalCloseExpanderIcon : finalOpenExpanderIcon)}
+        {({ isExpanded }) =>
+          isExpanded
+            ? React.cloneElement(finalCloseExpanderIcon, { className: iconClassName })
+            : React.cloneElement(finalOpenExpanderIcon, { className: iconClassName })
```

Comment:
Are you sure about this? It works fine without cloneElement if you pass the override.